cicd/_common_deploy_logic.sh

```diff
@@ -14,6 +15,7 @@ set -ex
 
 : ${COMPONENTS:=""}
 : ${COMPONENTS_W_RESOURCES:=""}
+: ${DEPLOY_TIMEOUT:="300"}
```

Comment:
Copy the COMPONENTS/COMPONENTS_W_RESOURCES method:

```suggestion
if [ ! -z "$DEPLOY_TIMEOUT" ]; then
    export TIMEOUT_ARG=$(-t $DEPLOY_TIMEOUT)
fi
```

Reply:
That approach is calling a func to do something a bit more complex -- I think the approach here works ok